plugins/data-management/src/HierarchicalTrackSelectorWidget/components/tree/TrackLabel.tsx

```diff
@@ -92,6 +106,16 @@ function TrackMenuButton({
       data-testid={`htsTrackEntryMenu-${id}`}
       menuItems={[
         ...(getSession(model).getTrackActionMenuItems?.(conf) || []),
+        {
+          label: model.isFavorite(conf)
```

Comment:
this is a nit but i might split this entirely into two different menu item objects configtional based on model.isFavorite since each branch of the object is conditional on it so the new code would be something like

```js
model.isFavorite(conf)?
    {label:'Remove from favorites',onClick:()=>model.removeFromFavorites(conf),icon:StartIcon}:
    {label:'Add to favorites',onClick:()=>model.addToFavorites(conf),icon:FilledStarIcon}
```